src/moar.h

```diff
@@ -40,11 +40,28 @@
  */
 #include <uv.h>
 
+#ifdef MVM_USE_C11_ATOMICS
+#include <stdatomic.h>
+typedef atomic_uintptr_t AO_t;
+#ifdef __clang__
+/* clang and gcc disagree on rvalue semantics of atomic types
+ * clang refuses to implicitly assign the value of an atomic variable to the
+ * regular non-atomic type. Hence we need this: */
+#define AO_READ(v) atomic_load_explicit(&(v), memory_order_relaxed)
```

Comment:
> Is there a downside to always using atomic_load_explicit, even when the compiler would allow for a plain read?

Well, I had thought "good question. I assume that it has no affect on the generated code, but it would make the C easier to read", so I tried it and tested what assembly language we got. And the only "downsides" are upsides. See commit 8a347416f56b9d6271045b24c904d1b38237f1bf - I hope that the CI tests like it too.